app/view_models/insights_view_model.rb

```diff
@@ -7,137 +7,110 @@ def upcoming_count
-      bidders.uniq.size
-    end.reduce(:+)
+  def published_auction_stat
+    {
+      statistic: published_auction_count,
+      label: 'total auctions',
+      href: 'chart-bids-by-auction'
+    }
   end
 
-  def total_delivery_time_length
-    total_time_length(accepted_auctions, 'accepted_at', 'ended_at')
+  def accepted_auction_stat
+    {
+      statistic: accepted_auctions_count,
+      label: 'successful delieries',
+      label_stat: "(#{accepted_auction_percent} success rate)"
+    }
   end
 
-  def total_auction_time_length
-    total_time_length(published_auctions, 'ended_at', 'started_at')
+  def unique_winners_stat
+    {
+      statistic: unique_auction_winners,
+      label: 'unique auction winners',
+      href: 'chart-winning-bid'
+    }
   end
 
-  def total_time_length(auctions, first_time, second_time)
-    auctions.map do |auction|
-      auction.send(first_time) - auction.send(second_time)
-    end.reduce(:+)
+  def average_bids_stat
+    {
+      statistic: AverageBidsPerAuction.new.to_s,
+      label: 'bids/auction',
+      href: 'chart4'
+    }
   end
 
-  def total_winning_bid_amount
-    completed_auctions
-      .map(&:lowest_bid)
-      .map(&:amount)
-      .reject(&:nil?)
-      .reduce(:+)
+  def unique_bidders_stat
+    {
+      statistic: UniqueBiddersPerAuction.new.to_s,
+      label: 'unique bidders per auction'
+    }
   end
 
-  def total_starting_price
-    completed_auctions.map(&:start_price).reduce(:+)
+  def vendors_with_bids_stat
+    {
+      statistic: UserQuery.new.with_bids.count,
+      href: 'agency-savings',
+      label: 'vendors who have placed bids'
+    }
   end
 
-  def published_auction_count
-    @_published_auction_count ||= published_auctions.count
+  def average_auction_length_stat
+    {
+      statistic: AverageAuctionLength.new.to_s,
+      href: 'auction-length',
+      label: 'average auction length'
+    }
   end
 
-  def completed_auction_count
-    @_completed_auction_count ||= completed_auctions.count
+  def average_winning_bid_stat
+    {
+      href: 'chart-winning-bid',
+      statistic: AverageWinningBid.new.to_s,
+      label: 'average winning bid'
+    }
   end
 
-  def completed_auctions
-    @_completed_auctions ||=
-      AuctionQuery
-      .new
-      .published
-      .delivery_due_at_expired
-      .with_bids_and_bidders
-      .where.not(bids: { id: nil })
+  def small_business_stat
+    {
+      statistic: UserQuery.new.small_business.count,
+      label: 'small businesses registered'
+    }
   end
 
-  def accepted_auctions
-    @_accepted_auctions ||= AuctionQuery.new.published.accepted
+  def unique_auction_winners
+    AuctionQuery.new.completed.map do |auction|
+      WinningBid.new(auction).find.bidder
+    end.uniq.count
   end
 
-  def published_auctions
-    @_published_auctions ||= AuctionQuery.new.published
+  def accepted_auction_percent
+    Percent.new(accepted_auctions_count, published_auction_count).to_s
```

Comment:
I still want this to be over `accepted_auctions_count + rejected_auctions_count` so it's just auctions we have ruled on